.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,70 @@
+# Originally sourced from GitHub with implicit lack of license
+
+name: Scorecard supply-chain security
+on:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '15 23 * * 0'
+  push:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      # SFUQUA: commented for now, probably nice to have later.
+      # id-token: write
+      # Uncomment the permissions below if installing in a private repository.
+      # contents: read
+      # actions: read
+
+    steps:
+      - name: "Checkout code"
```

Comment:
Although not important, name doesn't need to be quoted

Reply:
Agree. I copied and pasted from an example and meant to change that. I'm actually annoyed by unnecessary quotations in yml, so I'm going to change that.